provider-attributes.json

```diff
@@ -0,0 +1,676 @@
+      },
+      {
+        "key": "intel",
+        "description": "Intel"
+      },
+      {
+        "key": "xilinx",
+        "description": "Xilinx"
+      }
+    ]
+  },
+  "hardware-gpu-model": {
+    "key": "hardware-gpu-model",
+    "type": "option",
+    "required": false,
+    "description": "GPU model.",
+    "values": [
+      {
+        "key": "h100",
+        "description": "Nvidia h100"
+      },
+      {
+        "key": "a100",
+        "description": "Nvidia a100"
+      },
+      {
+        "key": "a40",
+        "description": "Nvidia a40"
+      },
+      {
+        "key": "a16",
+        "description": "Nvidia a16"
+      },
+      {
+        "key": "t4",
+        "description": "Nvidia t4"
+      },
+      {
+        "key": "v100",
+        "description": "Nvidia v100"
+      },
+      {
+        "key": "tesla-p4",
+        "description": "Nvidia tesla-p4"
+      },
+      {
+        "key": "tesla-p100",
+        "description": "Nvidia tesla-p100"
+      },
+      {
+        "key": "tesla-k80",
+        "description": "Nvidia tesla-k80"
+      },
+      {
+        "key": "rtx-3050",
+        "description": "Nvidia rtx-3050"
+      },
+      {
+        "key": "rtx-3060",
+        "description": "Nvidia rtx-3060"
+      },
+      {
+        "key": "rtx-3060-ti",
+        "description": "Nvidia rtx-3060-ti"
+      },
+      {
+        "key": "rtx-3070",
+        "description": "Nvidia rtx-3070"
+      },
+      {
+        "key": "rtx-3070-ti",
+        "description": "Nvidia rtx-3070-ti"
+      },
+      {
+        "key": "rtx-3080",
+        "description": "Nvidia rtx-3080"
+      },
+      {
+        "key": "rtx-3080-ti",
+        "description": "Nvidia rtx-3080-ti"
+      },
+      {
+        "key": "rtx-3090",
+        "description": "Nvidia rtx-3090"
+      },
+      {
+        "key": "rtx-3090-ti",
+        "description": "Nvidia rtx-3090-ti"
+      },
+      {
+        "key": "gtx-1050",
+        "description": "Nvidia gtx-1050"
+      },
+      {
+        "key": "gtx-1050-ti",
+        "description": "Nvidia gtx-1050-ti"
+      },
+      {
+        "key": "gtx-1060",
+        "description": "Nvidia gtx-1060"
+      },
+      {
+        "key": "gtx-1070",
+        "description": "Nvidia gtx-1070"
+      },
+      {
+        "key": "gtx-1070-ti",
+        "description": "Nvidia gtx-1070-ti"
+      },
+      {
+        "key": "gtx-1080",
+        "description": "Nvidia gtx-1080"
+      }
+    ]
+  },
+  "hardware-disk": {
+    "key": "hardware-disk",
+    "type": "option",
+    "required": true,
+    "description": "Storage type.",
+    "values": [
+      {
+        "key": "ssd",
+        "description": "ssd"
+      },
+      {
+        "key": "hdd",
+        "description": "hdd"
+      },
+      {
+        "key": "nvme",
+        "description": "nvme"
+      }
+    ]
+  },
+  "hardware-disk-size": {
+    "key": "hardware-disk-size",
+    "type": "number",
+    "required": false,
+    "description": "Storage size."
```

Comment:
Should we specify units? (ex: Bytes)

Reply:
Yes that's a good idea

Reply:
I think we should specify the size and type ie hdd/ssd/nvme

Reply:
> I think we should specify the size and type ie hdd/ssd/nvme

https://github.com/maxmaxlabs/cloudmos-config/pull/1#issuecomment-1653942001